docs/api-reference/next.config.js/rewrites.md

```diff
@@ -0,0 +1,112 @@
+---
+description: Add rewrites to your Next.js app.
+---
+
+# Rewrites
+
+Rewrites allow you to map an incoming request path to a different destination path.
+
+Rewrites are only available on the Node.js environment and do not affect client-side routing.
```

Comment:
This should read differently IMO, "To use rewrites to do client-side routing, use `<Link href="" as="">` ...." with link to docs.

Reply:
We can add that later, initially I feel it's good to be clear that the rewrites property does not adjust client-side routing in any way.